pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -472,3 +586,48 @@ func (r *ReconcileIBMBlockCSI) reconcileClusterRoleBinding(instance *ibmblockcsi
 	}
 	return nil
 }
+
+func (r *ReconcileIBMBlockCSI) deleteClusterRoleBindings(instance *ibmblockcsi.IBMBlockCSI) error {
+	logger := log.WithName("deleteClusterRoleBindings")
+
+	clusterRoleBindings := r.getClusterRoleBindings(instance)
+
+	for _, crb := range clusterRoleBindings {
+		found := &rbacv1.ClusterRoleBinding{}
+		err := r.client.Get(context.TODO(), types.NamespacedName{
+			Name:      crb.Name,
+			Namespace: crb.Namespace,
+		}, found)
+		if err != nil && errors.IsNotFound(err) {
+			continue
```

Comment:
Why don't we want to log the error in case we're not familiar with the error?

Reply:
see [previous conversation](https://github.com/IBM/ibm-block-csi-operator/pull/133#discussion_r586423233)